Add Navbar components

dukeph0 opened the pull request:
- Created a navigation bar component in the client folder
- Implemented Navbar using React Bootstrap
- Appears on Start and redirects to graph
- No analytics page yet to be redirected to
- Still need to implement a navbar in graphs  and analytics
- My comment is that some of the animations for the hovering feature will appear slow and the cards will have a gray color
- current look:
![image](https://user-images.githubusercontent.com/32912812/167308545-8b26ebc1-e1af-4958-b48b-b72d464931d7.png)

AaronGeesink commented on client/src/routes/Data.js:
```diff
@@ -3,6 +3,8 @@ import TwoSidedSlider from '../components/TwoSidedSlider';
 import {useEffect, useState} from 'react'
 import Card from '../components/Card';
 import AddCard from '../components/AddCard';
+import "bootstrap/dist/css/bootstrap.min.css";
```
I appears that importing `bootstrap/dist/css/bootstrap.min.css` messes with all of the CSS in the client. You might be able to go to that file and fix whatever is causing the CSS styling issue. If you can't pinpoint the problem, you can instead create a new file in the components folder called Navbar.css, set the CSS styling there, and import that instead.

AaronGeesink commented on client/src/components/navbar.js:
```diff
@@ -0,0 +1,43 @@
+    return (
+      <nav className="navbar navbar-expand-lg navbar-light bg-blue">
+      <a className="navbar-brand" href="/">
+        Home
+      </a>
+      <button
+        className="navbar-toggler"
+        type="button"
+        data-toggle="collapse"
+        data-target="#navbarNavDropdown"
+        aria-controls="navbarNavDropdown"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span className="navbar-toggler-icon"></span>
+      </button>
+
+      <div className="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul className="navbar-nav">
+          <li className="nav-item">
+            <a className="nav-link" href="graph">
+              View Data
+            </a>
+          </li>
+          <li className="nav-item">
+            <a className="nav-link" href="analytics">
+              Analytics
```
There are only supposed to be 2 tabs in the navbar. The "View Data" tab is the one with the sliders, and the "Analytics" tab is the one with the graphs. You can remove the "Home" tab, as we will not be using it.

AaronGeesink reviewed (changes requested):
The CSS file that you imported messed up the CSS for the rest of the page. Additionally, the names of the tabs are wrong. Please see the following comments for details on how to fix:

AaronGeesink commented:
Also, make sure to put the navbar at the top of graphs.js and see if you might have to change the navbar so that it looks good in all pages of the program. You should be able to add it to Graphs.js in the same way that you added it to data.js.

raymondwzeng reviewed (commented):
Assuming what Aaron mentioned gets resolved (or moved to upcoming sprint), looks good to me.

dukeph0 commented:
So I've added the navigation bar to analytics and corrected the tabs displayed on the navbar. However, the navbar on the homepage can't seem to appear in the correct area of the page. Creating another div ontop of <div className="page-content"> is not being allowed.

dukeph0 commented:
Navbar implemented for both the homepage and analytics
![image](https://user-images.githubusercontent.com/32912812/167380061-45ee5d6a-9354-43a4-9c28-faea7f62ddb1.png)
![image](https://user-images.githubusercontent.com/32912812/167380189-e22c658c-fc85-47bb-9e35-95780752e2ab.png)

raymondwzeng commented:
The most recent commit also fixes #70 as far as I know.
